### ISSUE TITLE
Emotes overlapped by read markers with maximized widgets

### ISSUE THREAD

**Issue by HarHarLinks** [vector-im/element-web#20942]
### Steps to reproduce

1. maximize widget
2. `/me ...`
3. have some people read it

### Outcome

![image](https://user-images.githubusercontent.com/2803622/152682144-531c3ea2-d1b9-4970-9ada-4fc27951a853.png)

### Operating system

arch

### Application version

Element Nightly version: 2022020601 Olm version: 3.2.8

### How did you install the app?

aur

### Will you send logs?

No

**Comment by HarHarLinks** [comment]
@SimonBrandner unless I should rather split out timestamps from this, I think this might not be as "O-Uncommon", certainly not as uncommon as someone using an emoji

**Comment by jryans** [comment]
Interesting, I suspect most people wouldn't encounter this, as you'd have to first enable "always show timestamps" for most of the overlapping to occur.

The read receipts do seem valuable to keep in some capacity, so we can't just remove them.

There's no immediately obvious solution, so I think we'd definitely need Design guidance here.

**Comment by HarHarLinks** [comment]
> first enable "always show timestamps"

are you certain this is not on by default? I tried to test logging into a fresh session on app.element.io and it was on. Unless it was saved in my profile earlier.

**Comment by jryans** [comment]
> are you certain this is not on by default? I tried to test logging into a fresh session on app.element.io and it was on. Unless it was saved in my profile earlier.

Yes, it is definitely off by default, as you can see [here](https://github.com/matrix-org/matrix-react-sdk/blob/f6565bfbc806434dafd18a3469ecf82b074488c6/src/settings/Settings.tsx#L457-L461). This particular setting is synced account wide, so it would apply to all of your Element Web sessions after changing it in any one of them.

**Comment by frakic** [comment]
I've also encountered this in video rooms with timestamps turned on:
![image](https://user-images.githubusercontent.com/18530109/169505577-79aff532-467e-4b58-a4ed-e639c3afe42a.png)
